rs/rust_canisters/backtrace_canister/backtrace_canister.did

```diff
@@ -0,0 +1 @@
+service : { ic0_trap : () -> (); oob : () -> (); unreachable : () -> () }
```

Comment:
This is a single line, should we format it to have each endpoint on a separate line?

Reply:
It's automatically generated by the `candid::export_service` macro and that's how they format it. So it would be a bit annoying for the test that asserts the candid file is correct if we were to modify the formatting here.